docs/rbac.md

```diff
@@ -43,7 +43,7 @@ _Note: The cluster-admin role is created by default in a Kubernetes cluster, so
 $ kubectl create -f rbac-config.yaml
 serviceaccount "tiller" created
 clusterrolebinding "tiller" created
-$ helm init --service-account tiller
+$ helm init --service-account tiller --max-history 200
```

Comment:
If you add it here then do you have to add it throughout the docs when `helm init` is mentioned? Should it be just added to the quick start?

Reply:
I added it to the quickstart because all the other examples of helm init are very specific but the quickstart is the most general.

Reply:
I think this is more the rbac doc than the quick start?